src/main.py

```diff
@@ -231,7 +231,7 @@ def run(self):
                                     sub.unsticky()
 
                                 print "Submitting game thread..."
-                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game"))
+                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game",self.TEAM_CODE))

```

Comment:
Actually it might be easier to switch, since there's no list of codes that MLB uses.